src/constants/Fonts.tsx

```diff
@@ -0,0 +1 @@
+export const FONTS = {}
```

Comment:
@marden-erin This is your first assignment from me! I would like you to find fonts we would like to use and put them here. Then test them out by making components in `src/components/simple-components/TextComponents.tsx`.

For now, you can put them on the `ExamplePage` to see them. I'm going to work on making a page on which we can put all of our components in the future. It'll be a nice place to reference them all!